bundles/org.openhab.binding.mecmeter/README.MD

```diff
@@ -0,0 +1,247 @@
+# MecMeter Binding
+
+This binding reads data from MEC power meter for providing electrical information for the electric circuit.
+
+To use this binding the meter must be installed, initialized and connected to the same network as openHAB. 
+
+## Supported Things
+
+The mecMeter is supported with firmware version starting from 2.0.0.
+There is exactly one supported thing type `meter`, which represents the electric meter.
+Its unique ID is the serial number.
+
+## Discovery
+
+MecMeters are automatically discovered via mDNS. 
+The IP of the Power Meter is automatically set and can be changed manually if needed.
+The default update interval is set to 5 seconds. Intervals from 1 to 300 seconds can be set manually.
+
+## Thing Configuration
+
+The thing has a few configuration parameters:  
+
+| Parameter       | Description                                                           |
+|-----------------|-----------------------------------------------------------------------|
+| password        | User-defined password during initializing the Power Meter.            |
```

Comment:
It may help to also document that this parameter is mandatory.

Reply:
done.